promotion/controllers.go

```diff
@@ -417,7 +417,7 @@ func CreatePromotion(service *Service) handlers.AppHandler {
 
 // ClobberedClaimsRequest holds the data needed to report claims that were clobbered by client bug
 type ClobberedClaimsRequest struct {
-	ClaimIDs []uuid.UUID `json:"claimIds" valid:"required"`
+	ClaimIDs []uuid.UUID `json:"claimIds" valid:"-"`
```

Comment:
I think now that your other PR is merged we should be fine leaving a validator on this?